### DISCUSSION TITLE
issue about get dns result from multi resolvers?

### DISCUSSION THREAD

**Discussion by Nyx2022** [projectdiscovery/dnsx#166] [General]
i found the logic in dnsx is send data to dns servers on :53 ,however ,i pass 100 resolvers to dnsx,but i got a list results that are in x.x.x.x/24,such as 10.10.10.2,10.10.10.10,10.10.10.20,etc，so is there any issue？because the real results of 100 resolvers are very differen

**Comment by ehsandeep** [comment]
@Nyx2022 Can you provide an example data/input/cmd to reproduce this issue?

**Reply by Nyx2022** [reply]
such as www.baidu.com, i got a result as following:
www.baidu.com [45.113.192.101]
www.baidu.com [45.113.192.102]

or 

www.baidu.com [104.193.88.77]
www.baidu.com [104.193.88.123]

no different result from different dns server when i use -r to pass a lsit of dns server from world,too

**Reply by muhammadismailkhan0009** [reply]
a domain/sub-domain can point to more than one ip-addresses due to multiple reasons, one being load balancing.
I used ```dig``` command for www.baidu.com on ```8.8.8.8``` server to get A records on ubuntu OS, i got the following A records: 45.113.192.101, 45.113.192.102.

**Comment by MetzinAround** [comment]
@ehsandeep do you think this is able to be closed? Seems like discussion dried up.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
